corehq/sql_accessors/sql_templates/soft_delete_cases.sql

```diff
@@ -1,14 +1,16 @@
-DROP FUNCTION IF EXISTS soft_delete_cases(TEXT, TEXT[], TIMESTAMP, TEXT);
+DROP FUNCTION IF EXISTS soft_delete_cases(TEXT, TEXT[], TIMESTAMP, TIMESTAMP, TEXT);
```

Comment:
i kept the signature in sync with the new parameter list, which is why i had to manually drop the functions in the migration